rtv/content.py

```diff
@@ -298,21 +298,41 @@ def __init__(self, name, submissions, loader=default_loader):
 
     @classmethod
     def from_name(cls, reddit, name, loader=default_loader):
+        
+        display_type = 'normal'
 
         if name == 'front':
             return cls('Front Page', reddit.get_front_page(limit=None), loader)
 
         if name == 'all':
             sub = reddit.get_subreddit(name)
-
+        
         else:
+            
+            if '/' in name:
+                parse = name.split('/')
```

Comment:
How about simplifying this to 
```
name, display_type = name.split('/')
```
Then you can get rid of the temporary parse variable all together